recording_external_revenue_account/models/recording_external_revenue.py

```diff
@@ -181,11 +212,17 @@ def _make_single_receivable_move_line(self, amount, due_date):
         return line
 
     def _map_receivable_account(self):
-        return self._partner_with_company_forced.property_account_receivable_id
+        return self.partner_id.property_account_receivable_id
 
     def _map_payment_term(self):
-        return self._partner_with_company_forced.property_payment_term_id
+        return self.partner_id.property_payment_term_id
 
-    @property
-    def _partner_with_company_forced(self):
-        return self.partner_id.with_context(force_company=self.company_id.id)
+    def _check_no_deprecated_journal_account(self, move):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/numigi/odoo-recording-addons/pullRequest?prid=5669413)